source/ada/lsp-ada_handlers.adb

```diff
@@ -341,9 +341,15 @@ package body LSP.Ada_Handlers is
          "iso-8859-1",
          Errors);
 
-      if not LSP.Types.Is_Empty (Errors.message) then
-         Self.Server.On_Show_Message (Errors);
-      end if;
+      --  We might have encountered errors when loading the project, for
```

Comment:
Can't we log something instead of reporting the messages? It might help for customer support

Reply:
Good idea, done in the new push!